packages/console/src/qr.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useRef, forwardRef } from 'react';
+import QRCode from 'react-qrcode-logo';
+
+interface StorachaQRCodeProps extends Partial<React.ComponentProps<typeof QRCode>> {
+  value: string;
+  onDownload?: (format?: 'png' | 'jpeg', filename?: string) => void;
+  downloadLabel?: string;
+}
+
+export const StorachaQRCode = forwardRef<QRCode, StorachaQRCodeProps>(({
+  value,
+  size = 256,
+  ecLevel = 'H',
+  fgColor = '#f70000',
```

Comment:
I think this should be the brand color - ideally it should just reference the Tailwind configs: https://github.com/storacha/upload-service/blob/07235198ede2388b3900f966c94a0157c3beb5ae/packages/console/tailwind.config.ts#L14